profiling/ldms-kokkos-sampler/Makefile

```diff
@@ -0,0 +1,32 @@
+
+
+OVIS_DIR=/projects/ovis/streams/opt/ovis
```

Comment:
The Kokkos sampler is not OVIS or LDMS specific so we should remove `OVIS` and `LDMS` references from the Makefile